catalog-rest-service/src/main/resources/ui/src/pages/IngestionPage/IngestionPage.component.tsx

```diff
@@ -121,14 +122,42 @@ const IngestionPage = () => {
     });
   };
 
-  const updateIngestionById = (
+  // code for updating ingestion with patch api
```

Comment:
Why are we keeping this commented code? If not required, should we clean up and remove it?

Reply:
@shahsank3t Yeah we can remove it.